### DISCUSSION TITLE
Ability to get a feed of Birdseye using /api/{camera_name}

### DISCUSSION THREAD

**Discussion by MrCaspan** [blakeblackshear/frigate#9193] [Ask A Question]
So I knwo from the [documentation ](https://docs.frigate.video/integrations/api/#get-apicamera_name) that I can pull a feed of a camera in  browser by doing:

`http://192.168.86.232:30058/api/HouseFront?fps=15&motion=1&h=1080`

Is there a way to do this for the Birdseye view? Or do I have to restream it as an RTSP feed?

I have tried the obvious with differnt case options with no luck:
`http://192.168.86.232:30058/api/Birdseye?fps=15&motion=1&h=1080`

**Comment by NickM-27** [comment]
Birdseye restream must be enabled for this

**Reply by MrCaspan** [reply]
Thanks for the quick reply! Sorry does this mean that if retream is enabeled that I can use it in the URL like I mentioned above `/api/birdseye/` or is it only available over RTSP?

**Reply by NickM-27** [reply]
you can access latest frame at /api/birdseye/latest.jpg and accessed in go2rtc via any streaming protocol it supports

**Reply by MrCaspan** [reply]
okay so NO you can't use it like the other cameras can for a live stream in your browser by just hitting a URL.. I have no clips as I only use Frigate for 24/7 recording of my cameras. All good I jsut thought this would be cool to have on my desktop when working from home.

Also the [documentation](https://docs.frigate.video/integrations/api/) for /api/ needs to be updated to include the export feature. This is what I was looking for when I found out about these other features!

**Reply by NickM-27** [reply]
you are looking at the stable docs, beta docs (export was added in beta) contain this info

**Reply by NickM-27** [reply]
>  All good I jsut thought this would be cool to have on my desktop when working from home.

right but there is no difference, go2rtc supports mjpeg output as well as many many other types

**Reply by MrCaspan** [reply]
its available in 0.12 stable no? im running 0.12.1-367D724 is that not the stable version? Or might this be the reason i am getting the error ` [concat @ 0x55c5d7ac5a80] No files to concat` when trying to do an export?

**Reply by NickM-27** [reply]
The ability to export a recording where frigate handles the combining of segments was added in 0.13, [here are the docs](https://deploy-preview-6262--frigate-docs.netlify.app/integrations/api#post-apiexportcamerastartstart-timestampendend-timestamp)

Seems you are likely thinking of some other api

**Reply by MrCaspan** [reply]
https://github.com/blakeblackshear/frigate/issues/2369#issuecomment-985141654  this post is from 2021.. it this still talking about 0.13 beta 2 years ago?

**Reply by MrCaspan** [reply]
It seems like there is now a NEW `/api/export/` feature in 0.13, but this thread talks about an endpoint that does this using a different API call in 0.12

here is a link to the comment that tells you how to use it. I can also see that you are on this thred :)
https://github.com/blakeblackshear/frigate/issues/2369#issuecomment-987531072

**Reply by NickM-27** [reply]
That api has a number of limitations like max number of segments and it is built in cache and returned via the api instead of being stored in the /exports folder. 

It is not really an export api just an api for notifications

**Reply by MrCaspan** [reply]
Im confused as this entire thread is about using this to export a custom clip but you are saying it should not be used or should be used for notifications? Why would you use this for notifications?

**Reply by NickM-27** [reply]
> Why would you use this for notifications?

because you want a notification to have a video of what happened, and this is the main way to directly request a video for an event.

However, it is suboptimal as the video is assembled in RAM and is not stored anywhere. It also has a maximum number of segments so any clip that is longer than an hour might fail.

The export api in 0.13 does not have any of these limitations

**Reply by MrCaspan** [reply]
Ahh sorry that makes much more sense that a notification allows you to kick off this export. SO i am 100% okay with creating this in RAM as i dont want it to live on the server so this is perfect. Ask for the segment and download it.. done. No clean up no nothing else.. 

Ill open an other ticket to discus the issue i am currently happening then. Sorry for all the questions!
